ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TransactionServiceAuthorizationIT.scala

```diff
@@ -83,6 +95,14 @@ class TransactionServiceAuthorizationIT extends LedgerTestSuite {
     "TXRejectMultiActorMissingAuth",
     "Reject exercising a multi-actor choice with missing authorizers",
     allocate(TwoParties, SingleParty),
+    tags = List(
```

Comment:
I wonder whether we can avoid injecting the security evidencing tags as arguments and instead add them as annotations on the tests methods. I think the separation of concerns here would make it clear that these are not affecting the scope of the test (like the other arguments such)